src/endpoints/cart.js

```diff
@@ -36,8 +36,8 @@ class CartEndpoint extends BaseExtend {
     return this.call
   }
 
-  AddProduct(productId, quantity = 1, data = {}) {
-    const body = buildCartItemData(productId, quantity)
+  AddProduct(productId, quantity = 1, data = {}, isSku) {
```

Comment:
For clarity, is this the only function we have an optional param like this for? Or have we done this with others?
Just clarifying if we're introducing a new pattern here.

Reply:
Yes, We just use `isSku` in `AddProduct`, and set the the `isSku = false` as default.